### PR TITLE
Create basic gui update - Lukas

### PR DESCRIPTION
**Pull request by Tacitor**
Adds the main menu that can exit and call a separate JFrame for study material.

### CODE REVIEW

**Comment by Tacitor** [comment]
Just a little bit of work.

File: KrampitzKreutzwiserICS4U/nbproject/project.properties [hunk 1]
```diff
@@ -54,7 +56,7 @@ javadoc.splitindex=true
 javadoc.use=true
 javadoc.version=false
 javadoc.windowtitle=
-main.class=krampitzkreutzwiserics4u.KrampitzKreutzwiserICS4U
+main.class=krampitzkreutzwiserics4u.MainMenu
```
Comment (Tacitor) [inline KrampitzKreutzwiserICS4U/nbproject/project.properties:59]: I renamed the main class to what we actually will use.

File: KrampitzKreutzwiserICS4U/src/krampitzkreutzwiserics4u/KrampitzKreutzwiserICS4U.java [hunk 2]
```diff
@@ -1,22 +0,0 @@
-/*
```
Comment (Tacitor) [inline KrampitzKreutzwiserICS4U/src/krampitzkreutzwiserics4u/KrampitzKreutzwiserICS4U.java:1]: poof! gone.

File: KrampitzKreutzwiserICS4U/src/krampitzkreutzwiserics4u/MainMenu.java [hunk 3]
```diff
@@ -0,0 +1,150 @@
+/*
```
Comment (Tacitor) [inline KrampitzKreutzwiserICS4U/src/krampitzkreutzwiserics4u/MainMenu.java:1]: Now a JFrame form

File: KrampitzKreutzwiserICS4U/src/krampitzkreutzwiserics4u/MaterialReview.java [hunk 4]
```diff
@@ -0,0 +1,98 @@
+/*
```
Comment (Tacitor) [inline KrampitzKreutzwiserICS4U/src/krampitzkreutzwiserics4u/MaterialReview.java:1]: The secondary window for the study material.

**Review by Tacitor — COMMENTED** [review]
All ready for you to approve @Evan-Kreutzwiser

**Review by Evan-Kreutzwiser — APPROVED** [review]

**Pull request merged**